src/main/java/bot/TaskType.java

```diff
@@ -0,0 +1,25 @@
+package bot;
+
+/**
+ * Types of Tasks
+ */
+public enum TaskType {
+
+    Deadline("D"), Event("E"), Todo("T"), Blank("B");
+
```

Comment:
This is quite cool. I did not know how to exactly use the functions in enum, so my work just used them like a pointer in C class.